src/widgets/breadcrumb/breadcrumb.tsx

```diff
@@ -153,6 +159,10 @@ const breadcrumb: BreadcrumbWidget = function breadcrumb(widgetParams) {
     ),
     link: cx(suit({ descendantName: 'link' }), userCssClasses.link),
   };
+  const templates = {
+    ...defaultTemplates,
+    ...userTemplates,
+  };
```

Comment:
this isn't needed normally, right?

Reply:
see https://github.com/algolia/instantsearch.js/pull/4785/commits/61ea462ec076c7fd7a38a33f8d0bde9344d72d0a